doc/examples.xml

```diff
@@ -529,6 +574,51 @@ true
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="RooksGraph">
+<ManSection>
+  <Oper Name="RooksGraph" Arg="[filt, ]m, n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>m</A> and <A>n</A> are positive integers, then this
+    operation returns the <E>rook's graph</E> for an <A>m</A> 
+    by <A>n</A> board.<P/>
+
+    From
+    <URL>https://mathworld.wolfram.com/RookGraph.html</URL>:
+    <P/>
+
+    <Q>The <C><A>m</A> x <A>m</A></C> rook's graph (confusingly called the 
+    <C><A>m</A> x <A>m</A></C> grid by Brouwer et al.1989, p. 440) and also
+    sometimes known as a lattice graph (e.g., Brouwer) is the graph Cartesian 
+    product <C>K_m square K_n</C> of complete graphs, which is
+    equivalent to the line graph <C>L(K_(m,n))</C> of the complete bipartite graph
+    <C>K_(m,n)</C>. This is the definition adopted for example by Brualdi and Ryser
+    (1991, p. 153), although restricted to the case <C>m = n</C>. This definition
+    corresponds to the connectivity graph of a rook chess piece (which can
```

Comment:
again I am not sure of what a connectivity graph is, I am aware of the term "graph connectivity" but not really of a connectivity graph :)

Reply:
I think I am going to have to re-write these sections in any case, since we might not be allowed to directly quote wolfram alpha!

Reply:
sounds good! :)